docs/arm64_host.md

```diff
@@ -0,0 +1,43 @@
+# Using EB corbos Linux SDK on arm64
+
+The EB corbos Linux SDK is primary developed and tested on amd64 Linux hosts,
+but it is also possible to use it on arm64 hosts.
+It was tested once on a Raspberry Pi 5 8GB successfully.
+
```

Comment:
The same steps as defined in https://elektrobit.github.io/ebcl_template/setup.html need to be performed first as well, maybe mention that for clarity

Reply:
Done.